docs/user-guide/organization/organization-init.md

```diff
@@ -0,0 +1,120 @@
+# Reference Architecture: Terraform AWS Organizations account baseline
+
+!!! help "How it works"
+    :books: [**documentation:** organization](../../how-it-works/organization/organization.md)
+    
+    :books: [**documentation:** organization accounts](../../how-it-works/organization/accounts.md)
+
+## User guide
+
+### Pre-requisites
+
+You'll need an email to [create and register your AWS Organization Root Account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
+For this purpose we recommend to avoid using an individual nominated email user. 
```

Comment:
The word "nominated" sounds wrong. How about "We recommend that you do not use a personal email account for this."?

Reply:
@diego-ojeda-binbash OJ thanks :) I'll update this 🤝 ✅